pkg/user/handler.go

```diff
@@ -23,22 +21,26 @@ import (
 	"github.com/gin-gonic/gin"
 )
 
-func NewHandler(hostname string, authentication config.Authentication, publicKey *rsa.PublicKey, userService userService, tokenService tokenService) Handler {
+func NewHandler(hostname string, accessTokenExpirationSeconds int, refreshTokenExpirationSeconds int, refreshTokenRememberMeExpirationSeconds int, publicKey *rsa.PublicKey, userService userService, tokenService tokenService) Handler {
 	return Handler{
 		hostname,
-		authentication,
+		accessTokenExpirationSeconds,
+		refreshTokenExpirationSeconds,
+		refreshTokenRememberMeExpirationSeconds,
 		publicKey,
 		userService,
 		tokenService,
 	}
 }
 
 type Handler struct {
-	hostname       string
-	authentication config.Authentication
-	publicKey      *rsa.PublicKey
-	userService    userService
-	tokenService   tokenService
+	hostname                                string
+	accessTokenExpirationSeconds            int
+	refreshTokenExpirationSeconds           int
+	refreshTokenRememberMeExpirationSeconds int
+	publicKey                               *rsa.PublicKey
```

Comment:
avoid using a pointer as we should not be able to mutate the public key here

Reply:
The reason for using a point is that rsa.GenerateKey, rsa.GenerateKey etc. all return pointers. You mentioned this before and if I recall we even have a Jira issue. It does make sense to prevent mutation so I created [this](https://github.com/dhis2-sre/im-manager/pull/724/commits/7ad16a65109033e9a40425d2acd6e148d3c088c1) commit. Please let me know what you think :slightly_smiling_face:

Reply:
As mentioned on Slack the commit has been removed and we'll deal with this later